LeetCodeSolutions/Program.cs

```diff
@@ -1,3 +1,22 @@
-﻿// See https://aka.ms/new-console-template for more information
-
-Console.WriteLine("Hello, World!");
\ No newline at end of file
+﻿namespace LeetCodeSolutions;
+public class Program
+{
+    public int Maximum69Number (int num)
+    {
+        if (num == 9999) return num;
+        var charArray = num.ToString().ToCharArray();
+        for(int i = 0;i<charArray.Length;i++){
```

Comment:
what other forms of the for loop would work here? do we really need to know the index?